.npmignore

```diff
@@ -0,0 +1,21 @@
+lib-cov
+*.seed
+*.log
+*.csv
+*.dat
+*.out
+*.pid
+*.gz
+
+pids
+logs
+results
+
+npm-debug.log
+node_modules
+coverage
+.jshintrc
```

Comment:
Copied from .gitignore, then adding a few things that probably shouldn't be in the tarfile sent to the npm repo.